docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,277 @@
+    recommended to precise the grants or fellowships given to some authors. One should in any case always acknowledge
+    the Astropy project, to which we are affiliated, and our mandatory external libraries (e.g. numpy, scipy, matplotlib,
+    iminuit).
+
+Contribution in conferences
+-----------------------------
+
+The section is about any contribution in conferences (a talk, a poster and their associated proceedings) related to
+the Gammapy project itself. It does not concern a technical or scientific work that uses Gammapy as an open library.
+In this case, the citation scheme of Gammapy should be used by these authors.
+
+As the length of the author list is generally a constrain, the author list is reduced to the short list of contributors
+for the conference, followed by 'The Gammapy team' associated with a link to the Gammapy team webpage:
+
+ ::
+
+  oo, ff, tt, for `the Gammapy team <https://gammapy.org/team.html>`_
+
+If there is a corresponding author, the 'Gammapy Coordination Committee' associated with its usual mailing list, is
+used. Concerning the acknowledgement, the Astropy project should always be mentioned, and if possible our mandatory
+external libraries.
+
+
+4. Metadata files
+==================================
+
+Depending on the software repository, different metadata files are used in the eco-system of Open Source research
+software and are mandatory.
+
+The file ``codemeta.json`` is used by HAL and Software Heritage, and recommended by
+`ESCAPE <https://escape2020.pages.in2p3.fr/wp3/eossr/v0.3.2/metadata.html>`_ or
+`EOSC <https://www.eosc-pillar.eu/news/illustrating-benefits-eosc-research-communities-germany>`_.
+
+The file ``CITATION.cff`` is used by Zenodo and GitHub. Its format should follow the rules set up by the
+`Citation File Format project <https://pythonlang.dev/repo/citation-file-format-citation-file-format/>`_. A GitHub
+Action can be used to automatically check the compliance with the latest format. At the date of this PIG, there is no
+official scheme to handle current and past affiliations, that might be needed for LTS releases for example (see the
+CFF project `Issue #268 <https://github.com/citation-file-format/citation-file-format/issues/268>`_). In the affiliation
+string, one could add a past affiliation as precised below, but with a risk that Zenodo or tools using ``CITATION.cff``
+to make the ``codemeta.json`` does not understand it.
+
+ ::
+
+    authors:
+      - family-names: XXX
+        given-names: "YYYY Z."
+        affiliation: "Lab AA, XX; Past: Lab BB, YY"
+        email: yyyy.xxx@gammapy.org
+        orcid: "https://orcid.org/0000-0000-0000-0000"
+
+The ``CITATION.cff`` file will be used as the main file to be maintained, while the ``codemeta.json`` will be
+automatically created. For any release, the former should be carefully updated and systematically a review should be
```

Comment:
Strike systematically. I'd also switch the order. One first reviews and then updates if necessary